osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -100,7 +100,8 @@ def extract_excerpts(execution_configuration, extraction_order, bbox_args, conve
                 if len(os.listdir(settings.RESULT_MEDIA_ROOT)) > 0:
                     for result_file_name in os.listdir(settings.RESULT_MEDIA_ROOT):
                         # gis files are packaged in a zip file
-                        if GisExcerptConverter.create_output_file(extraction_order, result_file_name):
+                        if GisExcerptConverter.create_output_file(
+                                extraction_order, result_file_name, export_format_key):
```

Comment:
It is already a function? I don't understand how your solution will look. Please make an example.